app/channel-settings/client/views/channelSettings.html

```diff
@@ -411,10 +411,10 @@ <h3 title="{{name}}" class="rc-user-info__name">{{> icon block="rc-header__icon"
 		<button class="rc-button rc-button--icon rc-button--outline js-edit" type="submit">{{> icon icon='edit'}}{{_ "Edit"}}</button>
 		{{/if}}
 		<button class="rc-button rc-button--icon rc-button--outline js-hide" type="submit">{{> icon icon='eye-off'}}{{_ "Hide"}}</button>
-		{{#if canLeaveRoom}}
-		<button class="rc-button rc-button--icon rc-button--outline rc-button--cancel js-leave">{{> icon icon='sign-out'}}{{_ 'Leave'}}</button>
-		{{/if}}
 	</div>
```

Comment:
This is hack to avoid the 'Leave channel' button exceeding the window width in German language.

Reply:
This shall be PRed to RC or the layout needs to adapted.